frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionsForm.tsx

```diff
@@ -19,13 +22,42 @@ const buildConfig = (name, networkType, typeParamsData): NetworkAttachmentDefini
     cniVersion: '0.3.1',
   };
 
-  _.forOwn(typeParamsData, (val, key) => {
-    config[key] = _.get(val, 'value', null);
-  });
+  let ipam = {};
+  try {
+    ipam = JSON.parse(_.get(typeParamsData, 'ipam.value', {}));
+  } catch (e) {
+    console.log(e); // eslint-disable-line no-console
```

Comment:
The user should be informed about unexpected format of data, the console logging is just optional way to handle that state.

In any case, the log message should be at least `info` level and provide additional info about context for debugging.

Reply:
Very good points. I'll address this in a follow-up.